config/routes.rb

```diff
@@ -4,6 +4,8 @@
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:create]
```

Comment:
- [ ] Once you add the new action method in the comments controller, allow the new route as part of this line.

Reply:
Thank you for letting me know. I changed this too. 👍